ui/utils.py

```diff
@@ -220,33 +220,33 @@ def generate_google_analytics_query(video_key):
     # https://developers.google.com/analytics/devguides/reporting/core/v3/reference
     START_DATE = '2005-01-01'
     END_DATE = '9999-01-01'
+    dimensions = [{'name': 'ga:eventAction'}]
+    if video.multiangle:
+        # [adorsk, 2018-03]
+        # Achtung! Only use the camera angle dimension for multiangle.
+        # Single-camera videos do not set a custom dimension for events
+        # sent to GoogleAnalytics.
+        dimensions.append({'name': 'ga:' + settings.GA_DIMENSION_CAMERA})
     query = {
         'reportRequests': [
             {
                 'viewId': settings.GA_VIEW_ID,
                 'dateRanges': [{'startDate': START_DATE, 'endDate': END_DATE}],
                 'metrics': [{'expression': 'ga:totalEvents'}],
-                'dimensions': [
-                    # [adorsk, 2018-03]
-                    # Achtung! a high degree of implicit coupling to
-                    # dimension names that have been manually set in GA.
-                    # Hard-coding for now.
-                    {'name': 'ga:eventAction'},
-                    {'name': 'ga:' + settings.GA_DIMENSION_CAMERA}
-                ],
+                'dimensions': dimensions,
                 'dimensionFilterClauses': [
                     {
                         'filters': [
                             {
                                 'dimensionName': 'ga:eventLabel',
                                 'operator': 'EXACT',
                                 # 2018-03, dorsk
-                                # Achtung! We do video_key.capitalize()
+                                # Achtung! We do str(video.key).capitalize()
                                 # because GA has capitalized event data,
                                 # due to an unexpected side-effect of the
                                 # react-ga library.
                                 # See: https://github.com/mitodl/odl-video-service/pull/472
-                                'expressions': [video_key.capitalize()]
+                                'expressions': [str(video.key).capitalize()]
```

Comment:
Change to `video.hexkey`

Reply:
@adorsk another minor issue, not sure if it's worth changing, is that the final column in the table is redundant for videos that aren't multi-angle:

<img width="441" alt="screen shot 2018-04-12 at 12 47 14 pm" src="https://user-images.githubusercontent.com/187676/38691775-c9ca44f0-3e4f-11e8-8b03-440ed0f2f106.png">